changelog.adoc

```diff
@@ -2,6 +2,22 @@
 
 // scriv-insert-here
 
+== 3.32.0 (2024-11-08)
+
+Bugfixes:
+
+* Remove the legacy Automate Transfer AP scope from Timers' scope dependencies.
+* Update `CURRENT_SCOPE_CONTRACT_VERSION` and Timers' `min_contract_version` from 1 to 2.
```

Comment:
I think we can phrase this in a user-facing way, rather than around the mechanics.

```suggestion
* Timers' scope data has changed. Users interacting with Timers will be prompted to login again.
```